src/styles/utils/variables.scss

```
@@ -0,0 +1,12 @@
+$box-size: 100px;
+$box-ruler: 7;
+$size-days: $box-ruler * $box-size + $box-ruler;
+
+$margin-left:
```

Use more obvious names;
```suggestion
$first-day-offset:
```

---

ready